### ISSUE TITLE
🐛 Bug Report: Amazon SES not working with appwrite

### ISSUE THREAD

**Issue by StruanMclean** [appwrite/appwrite#5324]
### 👟 Reproduction steps

I am using the provided credentials from amazon SES and its not working when i put it into the .env file.
When i put in one from sendinblue or sendgrid it works so it must be a problem with the credentials.

### 👍 Expected behavior

Appwrite SMTP docker container starts with no errors.

### 👎 Actual Behavior

When amazon SES creds are put in Appwrite SMTP docker container errors.

### 🎲 Appwrite version

Version 1.2.x

### 💻 Operating system

Linux

### 🧱 Your Environment

$6 ubuntu linux server from digitalocean.com. 
From market place here : [Marketplace](https://marketplace.digitalocean.com/apps/appwrite)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by StruanMclean** [comment]
It might be a error with how im getting the amazon SES creds and putting them into the .env file.

**Comment by joeyouss** [comment]
Hi
Can you please show us the error logs?

**Comment by Haimantika** [comment]
Hi @StruanMclean 👋 is there an update on this? Could you solve it?

> It might be a error with how im getting the amazon SES creds and putting them into the .env file.

**Comment by StruanMclean** [comment]
No sadly still not been able to solve the issue. ☹️

**Comment by Haimantika** [comment]
> No sadly still not been able to solve the issue. ☹️
 Like @joeyouss mentioned, can you show the error logs? Did you also try asking this question in our Discord channel? https://appwrite.io/discord

**Comment by StruanMclean** [comment]
appwrite-worker-mails  | Appwrite mails worker v1 has started
appwrite-worker-mails  | 
appwrite-worker-mails  | [notice] Starting worker 032e7adc10c9:7:v1-mails
appwrite-worker-mails  | Appwrite mails worker v1 has started
appwrite-worker-mails  | 
appwrite-worker-mails  | [notice] Starting worker 032e7adc10c9:7:v1-mails

**Comment by StruanMclean** [comment]
032e7adc10c9:7:v1-mails is not starting after putting in aws ses creds

**Comment by darshanthakral** [comment]
Is there any update on this issue? did we find some solution?

**Comment by StruanMclean** [comment]
> Is there any update on this issue? did we find some solution?

Still no fix.

**Comment by StruanMclean** [comment]
> Hi @StruanMclean I have taken this up with my team, will let you know about an update in a day soon 😄

Thanks for the help 👍🙂

**Comment by joeyouss** [comment]
Meanwhile, can you please show me the env vars you set? (you can hide the sensitive information), maybe some var might be missing and we can spot it

**Comment by StruanMclean** [comment]
My appwrite smtp works with brevo (Send in blue) but not with aws.
Its not a problem with how I'm putting them in.
I think its a problem with where I'm getting them from.
If you know where do i go in the aws panel to get the right type of creds.

Anything helps!

Kind regards,
Struan Mclean

**Comment by StruanMclean** [comment]
This is what I'm using but the creds don't look right 
compared to Brevo.
![Screenshot 2023-07-18 213020](https://github.com/appwrite/appwrite/assets/64728357/f2867209-13a5-4712-abaa-92cd5a7b637b)

**Comment by StruanMclean** [comment]
Aws creds,
SMTP username: 3c3n233nc2p......
SMTP password: wefefewfew....

Brevo creds
Email: ewwe...@gmail.com
SMTP password: wpeecccew.....

**Comment by StruanMclean** [comment]
Here are my creds in .env hidden info ![IMG20230718215359.jpg](https://github.com/appwrite/appwrite/assets/64728357/2488afea-9fba-44d3-9ff8-050f77b54a7b)

**Comment by StruanMclean** [comment]
Just tried Aws creds and it's broken my appwrite instance 😭

**Comment by stnguyen90** [comment]
@StruanMclean, 

> Here are my creds in .env hidden info ![IMG20230718215359.jpg](https://user-images.githubusercontent.com/64728357/254399653-2488afea-9fba-44d3-9ff8-050f77b54a7b.jpg)

Would you please try putting `tls` for `_APP_SMTP_SECURE` instead of `TLS`? If that doesn't work, try `ssl`.

Also, please make sure `_APP_SYSTEM_EMAIL_ADDRESS` is set to an email address that will work with your SMTP provider.

```
appwrite-worker-mails | Appwrite mails worker v1 has started
appwrite-worker-mails |
appwrite-worker-mails | [notice] Starting worker 032e7adc10c9:7:v1-mails
appwrite-worker-mails | Appwrite mails worker v1 has started
appwrite-worker-mails |
appwrite-worker-mails | [notice] Starting worker 032e7adc10c9:7:v1-mails
```

The logs should show something when an email is triggered. If there's nothing here, no email is being triggered. If you see a log about an email, but no error, Appwrite did not encounter an error and the problem might be with your SMTP provider or some spam filter blocking your email.

**Comment by darshanthakral** [comment]
@stnguyen90, 

I guess it's an internal bug from appwrite codebase because even I have tried configuring AWS SES in every possible combination with appwrite (Also with the way you shared). But it still doesn't work, And then to check I configured one of my other custom email from GoDaddy and it instantly worked right away. 

I think this is a serious issue because email support is a crucial part for any software product. I think the team should consider fixing it on priority and ASAP!
